src/core/cache-storage.ts

```diff
@@ -96,6 +96,10 @@ export class Cache {
             //ios safari 10.3 taints canvas with data urls unless crossOrigin is set to anonymous
             if (isInlineBase64Image(src) || useCORS) {
                 img.crossOrigin = 'anonymous';
+                // in chrome if the image loaded before without crossorigin it will be cached and used later even if the next usage has crossorigin
+                // so it will fail with CORS error, so add a random query parameter just to prevent the chrome from using the cached image
+                // see more info about the chrome issue in this link: https://stackoverflow.com/a/49503414
+                src = src + (src.indexOf('?') === -1 ? '?' : '&') + 'cacheBusting=1';
```

Comment:
It will only work on the first try, if the user execute again the process... It will fail agin. Here is a better approach:
```suggestion
                src = `${src}${src.endsWith('?') ? '&' : '?'}cacheBusting=${Math.random()}`;
```

Reply:
`endWith('?')` should be `includes('?')` 🤔